near/omni-bridge/src/lib.rs

```diff
@@ -198,6 +199,16 @@ impl FungibleTokenReceiver for Contract {
     ) -> PromiseOrValue<U128> {
         let parsed_msg: InitTransferMsg = serde_json::from_str(&msg).sdk_expect("ERR_PARSE_MSG");
         let token_id = env::predecessor_account_id();
+
+        // User has to pay for storage and we can't trust sender_id.
+        let signer_id = env::signer_account_id();
+
+        if self.acl_has_role(Role::NativeFeeRestricted.into(), signer_id.clone())
```

Comment:
Let's optimizing it a little bit and avoid extra storage read by verifying the native fee first